drop/setupDrop.py

```diff
@@ -24,14 +25,47 @@ def setupPaths(projectRoot):
     return repoPaths, projectPaths
 
 
-def installRPackages():
+def installRPackages(config: DropConfig = None):
     logger.info("check for missing R packages")
     script = Path(drop.__file__).parent / "installRPackages.R"
     requirements = Path(drop.__file__).parent / 'requirementsR.txt'
 
+    # install main packages
     response = subprocess.run(["Rscript", script, requirements], stderr=subprocess.STDOUT)
     response.check_returncode()
 
+    # install pipeline depending packages
```

Comment:
Do we want to include a test case to ensure that config-dependent installation works for different annotations? Might be something for the future and could blow up the test though.